Sources/WWCalendarTimeSelector.swift

```diff
@@ -999,7 +1033,11 @@ open class WWCalendarTimeSelector: UIViewController, UITableViewDelegate, UITabl
         case .multiple:
             del?.WWCalendarTimeSelectorDone?(picker, dates: multipleDates)
         case .range:
```

Comment:
Can you split enum value *range* to *dateRange* and *timeRange*, so we don't need *if* block in *swift* block

Reply:
done